robyn/openapi.py

```diff
@@ -0,0 +1,267 @@
+import json
+import os
+import re
+from inspect import Signature
+from typing import List, Dict, Union, Any, TypedDict
+
+import robyn
+
+
+class OpenAPI:
+    def __init__(
+        self,
+        openapi_title: str = "Robyn",
+        openapi_summary: str = None,
+        openapi_description: str = None,
+        openapi_terms_of_service: str = None,
+        openapi_version: str = None,
+        openapi_contact_name: str = None,
+        openapi_contact_email: str = None,
+        openapi_contact_url: str = None,
+        openapi_license_name: str = None,
+        openapi_license_url: str = None,
+        openapi_servers: List[Dict[str, Union[str, Any]]] = None,
+        openapi_external_docs: Dict[str, str] = None,
+        openapi_component_schemas=None,
+        openapi_component_responses=None,
+        openapi_component_parameters=None,
+        openapi_component_examples=None,
+        openapi_component_request_bodies=None,
+        openapi_component_security_schemes=None,
+        openapi_component_links=None,
+        openapi_component_callbacks=None,
+        openapi_component_path_items=None,
+    ):
+        self.openapi_schema = self.build_schema(
+            title=openapi_title,
+            summary=openapi_summary,
+            description=openapi_description,
+            terms_of_service=openapi_terms_of_service,
+            version=openapi_version,
+            contact_name=openapi_contact_name,
+            contact_email=openapi_contact_email,
+            contact_url=openapi_contact_url,
+            license_name=openapi_license_name,
+            license_url=openapi_license_url,
+            servers=openapi_servers,
+            external_docs=openapi_external_docs,
+            component_schemas=openapi_component_schemas,
+            component_responses=openapi_component_responses,
+            component_parameters=openapi_component_parameters,
+            component_examples=openapi_component_examples,
+            component_request_bodies=openapi_component_request_bodies,
+            component_security_schemes=openapi_component_security_schemes,
+            component_links=openapi_component_links,
+            component_callbacks=openapi_component_callbacks,
+            component_path_items=openapi_component_path_items,
+        )
+
+    def build_schema(
```

Comment:
please add docstrings here.

Reply:
what? isn't it already added for this function?